utils/utils.go

```diff
@@ -226,3 +227,16 @@ func IsDatabaseFile(path string) bool {
 	dbPath := config.Config().DatabaseLocation
 	return path == dbPath || path == dbPath+"-wal" || path == dbPath+"-shm"
 }
+
+func HumanReadableByteCountSI(b int64) string {
```

Comment:
i dont like this, i prefer if it just outputs the bytes with commas, like every other place in gb that a byte count is outputted

Reply:
see usages of `utils.FormatCommas`